py/torch_tensorrt/dynamo/conversion/aten_ops_converters.py

```diff
@@ -2610,6 +2610,30 @@ def aten_ops_pixel_unshuffle(
     )
 
 
+@dynamo_tensorrt_converter(torch.ops.aten.resize.default)
```

Comment:
Is there `torch.ops.aten.resize.default` in the [schema](https://github.com/pytorch/pytorch/blob/main/aten/src/ATen/native/native_functions.yaml)? I didn't find it. Can you double check?

Reply:
Typically, we only register the functions starting with `- func: ...` in the schema.
Can you try only registering `resize_`, whether the unit test with `resize` is able to pass?
If pass, I think only `resize_` is enough.